### DISCUSSION TITLE
2022/02/25/%E5%A4%9A%E8%AF%AD%E8%A8%80%E6%8F%92%E4%BB%B6%E6%B5%8B%E8%AF%95/

### DISCUSSION THREAD

**Discussion by giscus** [Westlifers/Westlifers.github.io#16] [Announcements]
# 2022/02/25/%E5%A4%9A%E8%AF%AD%E8%A8%80%E6%8F%92%E4%BB%B6%E6%B5%8B%E8%AF%95/

https://hexo.yougi.top/2022/02/25/%E5%A4%9A%E8%AF%AD%E8%A8%80%E6%8F%92%E4%BB%B6%E6%B5%8B%E8%AF%95/

**Comment by Westlifers** [comment]
将就，但是翻译功能迟迟无法上线，难度大。
